modules/proxy/proxy_test.go

```diff
@@ -0,0 +1,49 @@
+package proxy
```

Comment:
Copy right header is necessary for linter.

Reply:
updated.